src/main.js

```diff
@@ -6,8 +6,9 @@ import globalComponents from './plugins/globalComponents';
 import i18n from '@/plugins/vue-i18n';
 import './api/apiConfig';
 import './assets/tailwind.css';
+import ElementUI from 'element-ui'
 
-
+Vue.use(ElementUI);
```

Comment:
remove this line.
Components from  element ui should imported like this
![image](https://user-images.githubusercontent.com/9899327/130411685-3c90a547-d223-47a1-8a19-987420318690.png)